desi_randoms.py

```diff
@@ -0,0 +1,27 @@
+import numpy         as     np
+
+from   astropy.table import Table, unique
+from   ros_tools     import tile2rosette, calc_rosr
+
+
+def desi_randoms():
+    rand = Table.read('/global/cfs/cdirs/desi/survey/catalogs/SV3/LSS/random0/rancomb_brightwdup_Alltiles.fits')
+    # rand.pprint()
+
+    rand             = unique(rand, keys='TARGETID')
+    rand['ROS']      = tile2rosette(rand['TILEID'])
+    rand['ROS_DIST'] = 1.e99
+
+    for rosn in np.unique(rand['ROS']):
+        isin = (rand['ROS'].data == rosn)
+
+        new_dist = calc_rosr(rosn, rand['RA'][isin], rand['DEC'][isin])
+    
+        rand['ROS_DIST'][isin] = np.minimum(rand['ROS_DIST'][isin], new_dist)
```

Comment:
Think you need the ROS_DIST > 0.5 & < 1.5 pushed here before I merge?

Reply:
Very good.  That should be in.